pliers/tests/extractors/test_model_extractors.py

```diff
@@ -447,3 +448,20 @@ def compute_expected_length(stim, ext):
     with pytest.raises(ValueError) as err:
         AudiosetLabelExtractor(top_n=10, labels=labels)
     assert 'Top_n and labels are mutually exclusive' in str(err.value)
+
+def test_spice_extractor():
+    audio_stim = AudioStim(join(AUDIO_DIR, 'homer.wav'))
+    audio_filter = AudioResamplingFilter(target_sr=16000)
+    audio_resampled = audio_filter.transform(audio_stim)
+
+    ext = TFHubAudioExtractor(SPICE_URL, load_type="saved_model")
+    r_orig = ext.transform(audio_stim).to_df()
+    assert r_orig.shape == (74, 6)
+
+    r_orig.onset.min() == 0.0
+    r_orig.duration.min() == r_orig.duration.max() == 0.04594594594594594
+    r_orig.uncertainty[0] == 0.974131
+    r_orig.pitch[0] == 0.171392
```

Comment:
more relevant for potential Neuroscout applications than for this: the Spice docs say something about calibrating output values: https://tfhub.dev/google/spice/2

Reply:
Is this for obtaining actual pitch values? Since this is just a scale-type transformation I'm not sure it makes much of a difference for fMRI models actually

Reply:
good point - I'll resolve this.